package.json

```diff
@@ -30,16 +30,19 @@
     "lint": "tslint -t stylish --project \"tsconfig.json\"",
     "test": "npm run test-only",
     "test-only": "jest --coverage",
-    "watch": "tsc -w -p tsconfig.build.json"
+    "watch": "tsc -w -p tsconfig.build.json",
+    "webpack": "webpack"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@types/bezier-js": "^0.0.7",
```

Comment:
How production ready is bezier-js? Are there any concerns about using this library?

Reply:
Seems pretty legit, the things documented here http://pomax.github.io/bezierjs/ don't seem to have significant active issues (issues are mostly feature requests)